backend/src/apiserver/server/util.go

```diff
@@ -62,6 +62,10 @@ func isYamlFile(fileName string) bool {
 	return strings.HasSuffix(fileName, ".yaml") || strings.HasSuffix(fileName, ".yml")
 }
 
+func isPipelineYamlFile(fileName string) bool {
+	return strings.HasSuffix(fileName, "pipeline.yaml")
```

Comment:
should it be exact match or suffix?

Reply:
I'm OK with either.
Suffix can be more convenient in obscure case where we want to have multiple pipeline files in the same directory and then pack each of them in an archive. The test pipeline packages contained named pipelines, so that probably affected my decision.
The problem is that suffix can match multiple files in an archive.
Probably you're right that I should stick with the exact match.
@neuromage , @hongye-sun @gaoning777 WDYT?

Reply:
Exact match seems better to me. You also want to update your error message below to match (i.e., strip asterisk in *pipeline.yaml)

Reply:
Done.